api/api.yaml

```diff
@@ -132,6 +132,26 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/routers:
+    post:
+      tags: ['Control']
+      operationId: set_protocol_state
+      description: >-
+        Updates the state of the device protocol peer.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../control/control.yaml#/components/schemas/Protocol.State'
+      responses:
+        '200':
+          $ref: '../result/request.yaml#/components/responses/Success'
```

Comment:
The response is not flexible enough to indicate what "names" failed to start/stop.  The response should be more detailed to allow users to determine if they want the test to continue or fail in the case where the protocol state isn't met.

Reply:
this makes more sense if regex is used in request names.